app/models/import.rb

```diff
@@ -0,0 +1,55 @@
+class Import < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :organization
+
+  validates :user, presence: true
+  validates :organization, presence: true
+
+  def self.retrieve_wishlist_data(url)
+    conn = Faraday.new(:url => url) do |faraday|
+      faraday.use FaradayMiddleware::FollowRedirects
+      faraday.request  :url_encoded             # form-encode POST params
+      faraday.response :logger                  # log requests to STDOUT
+      faraday.adapter  Faraday.default_adapter  # make requests with Net::HTTP
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.